client/components/editor.vue

```diff
@@ -187,11 +187,11 @@
   import {cursorsPlugin} from './utils/cursors-plugin';
   import {commentPlugin} from './utils/comment-plugin';
   import {titleSizePlugin} from './utils/title-size-plugin.js';
-  import addCommentPlugin, {addHighlight, removeHighlight, updateChunks} from './utils/add-comment-plugin';
+  import addCommentPlugin, {addHighlight, removeHighlight} from './utils/add-comment-plugin';
   import {toggleLink, clearLink} from './utils/link.js';
-  import {Snackbar} from '../snackbar';
 
   const mac = typeof navigator !== 'undefined' ? /Mac/.test(navigator.platform) : false;
+  const highlightIds = new Map();
```

Comment:
Why is this not per component?

Reply:
Also, name it so that it is clear what the map contains.

Reply:
`highlightIdsToCommentIds`

Reply:
Done, moved to component.